src/uproot/behaviors/TH1.py

```diff
@@ -229,8 +294,7 @@ def axis(self, axis=0):  # default axis for one-dimensional is intentional
 
     @property
     def weighted(self):
-        sumw2 = self.member("fSumw2", none_if_missing=True)
-        return sumw2 is not None and len(sumw2) == self.member("fNcells")
+        return super().weighted
```

Comment:
Since the subclass's implementation is exactly the same as the superclass's, the whole definition can be skipped. Some linters would probably require that.

I can't delete this in a suggestion through the web form because it includes deleted lines, but I recommend it. I'll use the tag "unnecessary subclass method implementation" for all cases like this one.

Reply:
> Since the subclass's implementation is exactly the same as the superclass's, the whole definition can be skipped. Some linters would probably require that.
> 
> I can't delete this in a suggestion through the web form because it includes deleted lines, but I recommend it. I'll use the tag "unnecessary subclass method implementation" for all cases like this one.

This was my first approach but it failed, I think due to `no_inherit = (uproot.behaviors.TH1.TH1,)`. Probably because I was at first implementing the methods in `TF1`, I assumed it wouldn't also work if I implemented them in `Histogram` but I was wrong, thanks!

Reply:
Oh, right! I forgot about that. The issue is that ROOT's histogram inheritance tree is... surprising... and we used to have a problem with inherited methods being the wrong implementation. (So, for instance, if we hadn't thought about some subclass like TProfile yet, all of these methods would appear to be implemented, but they would do the wrong things.) The `no_inherit` mechanism is to prevent inheritance, which we opt into intentionally. (So an unimplemented subclass appears to be unimplemented.)

Calling `super().function(all, of, the, arguments)` is probably the best way to do it. Go ahead and resolve all of those issues.